app/assets/javascripts/alchemy/alchemy.link_dialog.js.coffee

```diff
@@ -20,12 +20,19 @@ class window.Alchemy.LinkDialog extends Alchemy.Dialog
     # Store some jQuery objects for further reference
     @$page_anchor = $('#page_anchor', @dialog_body)
     @$internal_urlname = $('#internal_urlname', @dialog_body)
-    @$internal_anchor = $('#internal_anchor', @dialog_body)
     @$external_url = $('#external_url', @dialog_body)
     @$public_filename = $('#public_filename', @dialog_body)
     @$overlay_tabs = $('#overlay_tabs', @dialog_body)
     @$page_container = $('#page_selector_container')
-    @initInternalAnchors()
+    @$internal_links = new Alchemy.InternalLinksSelect
```

Comment:
Ah. And please remove the `$` prefix from the variable name, because we don't store a jQuery object into it anymore.